src/change_stream/option.rs

```diff
@@ -0,0 +1,86 @@
+use serde::{Deserialize, Serialize};
```

Comment:
ditto above -- this file should have a comment on top with a description of the module (see `src/event/mod.rs` for an example).

Reply:
fixed